frontend/src/metabase/collections/components/PinnedItemCard/PinnedItemCard.stories.tsx

```diff
@@ -18,8 +22,14 @@ const collection = {
 const onCopy = action("onCopy");
 const onMove = action("onMove");
 
+const store = getStore(mainReducers, createMockState());
```

Comment:
I don't think we want to have store-connected components in the Storybook. @ranquild, can you please confirm?

I believe the proper fix is to refactor `ActionMenu` so that it's not connected to the store.

Reply:
Agree, we don't want redux-based components here.